app.rb

```diff
@@ -0,0 +1,115 @@
+require_relative 'rental'
+require_relative 'person'
+require_relative 'classroom'
+require_relative 'book'
+require_relative 'student'
+require_relative 'teacher'
+
+class App
```

Comment:
[OPTIONAL] Kindly remove the test code in the `person.rb` file because it is interfering with the basic UI. 

![image](https://user-images.githubusercontent.com/38088818/211133848-2779493b-46d5-43d8-bb6d-39c8d41325bd.png)

![image](https://user-images.githubusercontent.com/38088818/211133866-d2595142-e4b3-4c68-83bc-d71bb252612e.png)

Reply:
Fixed.